tests/regression

```diff
@@ -23,7 +23,7 @@
 
 GLBL_ARCH_LE_SUPPORT=" \
 	x86 x86_64 x32 \
-	arm aarch64 \
+	arm aarch64 loongarch64 \
```

Comment:
Better to start a newline, every line seems to only contain similar architectures.